js/test/strings-ext/testlistfmt_be_BY.js

```diff
@@ -0,0 +1,253 @@
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два"]), "адзін два");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatThreeShort: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры"]), "адзін два тры");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatFourShort: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры", "чатыры"]), "адзін два тры чатыры");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatFiveShort: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры", "чатыры", "ხუთი"]), "адзін два тры чатыры ხუთი");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatOneFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін"]), "адзін");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatTwoFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два"]), "адзін два");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatThreeFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры"]), "адзін два тры");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatFourFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры", "чатыры"]), "адзін два тры чатыры");
+        test.done();
+    },
+    testListFmtUnitStylekaGENumberFormatFiveFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры", "чатыры", "ხუთი"]), "адзін два тры чатыры ხუთი");
```

Comment:
ხუთი -> not cyrillic characters. Where did that come from?

Reply:
Probably I missed it when I copied texts. I updated it from the google translate.